.github/actions/nss/action.yml

```diff
@@ -16,35 +16,68 @@ inputs:
 runs:
   using: composite
   steps:
+    - name: Check system NSS version
+      shell: bash
+      run: |
+        if ! command -v pkg-config &> /dev/null; then
+          echo "BUILD_NSS=1" >> "$GITHUB_ENV"
+          exit 0
+        fi
```

Comment:
For testing the availability of commands, I use `hash pkg-config 2>/dev/null`.  YMMV.  Note that just invoking pkg-config without testing for it first will also achieve the same:

```suggestion
```

Reply:
Just invoking it if it isn't installed will fail with an error code != 0 and so stop CI.